src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -171,13 +170,17 @@ func useGoodOldConfig(configDir, configPath string) bool {
 // Init initializes the configuration system.
 func Init(version string) (*Config, error) {
 	file := "lantern-" + version + ".yaml"
-	configDir, configPath, err := InConfigDir(file)
+	_, configPath, err := InConfigDir(file)
 	if err != nil {
 		log.Errorf("Could not get config path? %v", err)
 		return nil, err
 	}
-	ok := useGoodOldConfig(configDir, configPath)
-	if !ok {
+	run := isGoodConfig(configPath)
```

Comment:
We'd better to respect existing config file as it still contains user settings. We can add the check for cloudfront masquerades in `isGoodConfig` so 2.0.7 will  drop config of previous versions, but later versions can keep user settings.

If we do decide to drop previous config, function `useGoodOldConfig` can be removed.